src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -229,24 +229,36 @@ const GroupDetails = createReactClass({
         default:
           return <LoadingError onRetry={this.remountComponent} />;
       }
-    } else if (this.state.loading || !group) return <LoadingIndicator />;
+    }
+
+    const isLoading = loading || !group;
 
     return (
       <Feature features={['sentry10']}>
-        {({hasFeature}) => (
-          <React.Fragment>
-            {hasFeature &&
-              showGlobalHeader && (
+        {({hasFeature: hasSentry10}) => {
+          const shouldShowGlobalheader = hasSentry10 && showGlobalHeader;
+          return (
+            <React.Fragment>
+              {shouldShowGlobalheader && (
                 <GlobalSelectionHeader
                   organization={organization}
                   forceProject={project}
                 />
               )}
-            {hasFeature &&
-              showGlobalHeader && <PageContent>{this.renderContent()}</PageContent>}
-            {!hasFeature && this.renderContent()}
-          </React.Fragment>
-        )}
+              {isLoading ? (
+                <LoadingIndicator />
+              ) : (
+                <React.Fragment>
```

Comment:
I don't think you need to use `Fragment` here:

```js
isLoading ? (
  <LoadingIndicator />
) : shouldShowGlobalHeader ? (
    <PageContent>{this.renderContent()}</PageContent>
  ) : (
    this.renderContent()
  )
)
```
If you wanted to avoid the double-ternary you could move the second part into a function called `getContentWithContainer` or something:

```js
isLoading ? <LoadingIndicator /> : this.getContentWithContainer()
```